resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -28,10 +28,17 @@ enum PointerSize {
     ExtraLarge
 }
 
+enum BinarySettingState {
+    KeepCurrentValue
+	Enabled
+	Disabled
+}

```

Comment:
```suggestion
enum Status {
        KeepCurrentValue
	Enabled
	Disabled
}

```

Reply:
I thought about this and I think the name `status` feels more natural to me than `BinarySettingState`

Reply:
I disagree - `Status` is too generic.